### ISSUE TITLE
In activity detail, button favorite and 3 dots menu have no accessible name

### ISSUE THREAD

**Issue by akhanfir** [Meeds-io/meeds#1376]
**Steps to reproduce**

1. Create a space

2. Create an activity

3. Open DevTools of your browser

4. In tab lighthouse, launche accessiblity check

**Current behaviour**

Button favorite and 3 dot menu button have not accessible name.

**Expected behavior**

Both button must have an attribute "aria-label". 

For favorite, If the element is not in fav, the label must but "Bookmark it to easily find it when searching" 

If the element is already in favorite, the label must be "Remove from favorites"

It is the same text as the tooltips.
It must be localized.

For the activity 3 dots menu, the label must be "Open activity menu"
